src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -114,18 +118,26 @@ class OrganizationMembersWrapper extends AsyncView<Props> {
     return count ? count.toString() : null;
   }
 
-  updateRequestList = (id: string) =>
+  removeAccessRequest = (id: string) =>
     this.setState(state => ({
-      requestList: state.requestList.filter(({id: existingId}) => existingId !== id),
+      requestList: state.requestList.filter(m => m.id !== id),
```

Comment:
I'm not sure if the single character name is better than the previous destructure. Can we give it a better name?

Reply:
Yeah I'll fix this up in a separate PR, sounds good.